docs/Release_Guide/release_steps/metviewer/update_version_official.rst

```diff
@@ -0,0 +1,17 @@
+Update Version Number for Release
+---------------------------------
+
+Remove **-dev** from the version number:
+
+* We are naming releases with X.Y.Z format even if Z is 0.
+* The file containing the version number is located at docs/version.
+* In the METviewer/build.xml file, assign the version in the 'dist' target
+* In the METviewer/vebapp/metviewer/metviewer1.jsp file, assign the version to <div id='release'> and <title>
+* In the develop branch, the version should match the upcoming release with -dev added to the end like X.Y.Z-betaN-dev, i.e. 4.0.0-beta1-dev
+* Remove **-dev** from the version number so that it matches the release you are creating.
+
+Update the version number in the quick search links:
```

Comment:
This line should be removed as there is no docs/Users_Guide/quicksearch.rst for METviewer documentation.